Tasks/Maven/task.json

```diff
@@ -105,7 +105,26 @@
             "groupName": "codeCoverage",
             "helpMarkDown": "Comma separated list of filters to include or exclude classes from collecting code coverage. For example, +:com.*,+:org.*,-:my.app*.*. Refer http://www.eclemma.org/jacoco/trunk/doc/maven.html.",
             "visibleRule": "codeCoverageTool = JaCoCo"
-        },   
+        }, 
+        {
+            "name": "classFilesDirectories",
+            "type": "string",
+            "label": "Class Files Directories",
+            "required": false,
+            "groupName": "codeCoverage",
+            "helpMarkDown": "This field is required for a multi module project. Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. For example, classes1/Tests,classes2/Tests2. Refer https://dzone.com/articles/jacoco-maven-multi-module",
```

Comment:
Please also Mention that This will require Ant to be there. Also Please confirm text in help from gopi. dzone.com link does not look correct - there should be a official equivalent here